src/Cardano/Wallet/Api/Types.hs

```diff
@@ -101,6 +108,14 @@ import qualified Data.Text.Encoding as T
                                   API Types
 -------------------------------------------------------------------------------}
 
+data Address = Address
+    { _id :: !(ApiT P.Address)
+    , _state :: !AddressState
+    } deriving (Eq, Generic, Show)
+
+data AddressState = Used | Unused
+    deriving (Eq, Generic, Show)
```

Comment:
That, we want in our `Primitive.Model` the same way we have other metadata about various core types.

Reply:
Done.